CRM/Report/Form/ActivitySummary.php

```diff
@@ -355,12 +350,7 @@ public function from($durationMode = FALSE) {
               {$this->_aclFrom}";
     }
 
-    if ($this->_emailField) {
-      $this->_from .= "
-            LEFT JOIN civicrm_email  {$this->_aliases['civicrm_email']}
-                   ON {$this->_aliases['civicrm_contact']}.id = {$this->_aliases['civicrm_email']}.contact_id AND
-                     {$this->_aliases['civicrm_email']}.is_primary = 1 ";
-    }
+    $this->joinEmailFromClause();
```

Comment:
same as above

Reply:
ug I took from the wrong version! That was one of the things I fixed when I did my cut 
https://github.com/civicrm/civicrm-core/pull/11814/files#diff-35417c3a249b354869b9e25ca48e9d5cR585